packages/cli-plugin-scaffold-app-graphql/index.js

```diff
@@ -0,0 +1,249 @@
+	{
+		name: "cli-plugin-scaffold-template-graphql-app",
+		type: "cli-plugin-scaffold-template",
+		scaffold: {
+			name: "GraphQL Apollo Service App",
+			questions: () => {
+				return [
+					{
+						name: "apiLocation",
+						message: "Enter existing GraphQL API location (relative to the project root) or empty for none",
+						validate: apiLocation => {
+							if(!apiLocation) {
+								return true;
+							}
+							if(fs.existsSync(apiLocation) === false) {
+								return "There is no GraphQL API in given location";
+							}
+							try {
+								findDataModels(apiLocation);
+							}
+							catch (ex) {
+								return `Could not find existing API model in ${apiLocation}, error: ${ex.message}`;
+							}
+							return true;
+						}
+					},
+					{
+						name: "existingDataModelName",
+						message: "Choose data model to use",
+						type: "list",
+						when: ({apiLocation}) => {
+							return !!apiLocation;
+						},
+						choices: ({apiLocation}) => {
+							const names = findDataModels(apiLocation);
+							return names.map(({modelName}) => modelName);
+						},
+						validate: (name, {apiLocation}) => {
+							if(!name) {
+								return "Please enter a data model name";
+							}
+							else if(!name.match(/^([a-z]+)$/i)) {
+								return "A valid entity name must consist of letters only.";
+							}
+							try {
+								const names = findDataModels(apiLocation).map(({modelName}) => modelName);
+								if(!names.includes(name)) {
+									throw new Error();
+								}
+								return true;
+							}
+							catch (ex) {
+								return `A data model with name "${name}" does not exist`;
+							}
+						}
+					},
+					{
+						name: "packageLocation",
+						message: `Enter package location (will be located in "${appsPluginsLocation}" directory)`,
+						default: "books",
+						validate: name => {
+							if(!name) {
+								return "Please enter a package location";
+							}
+							const packageLocation = createPackageLocation(name);
+							
+							const locationFullPath = path.resolve(packageLocation);
+							if(fs.existsSync(locationFullPath)) {
+								return `The target location already exists: ${packageLocation}`;
+							}
+							
+							const rootResourcesPath = findUp.sync("resources.js", {
+								cwd: locationFullPath
+							});
+							
+							if(!rootResourcesPath) {
+								return `Resources file was not found. Make sure your package is inside of your project's root and that either it or one of its parent directories contains resources.js`;
+							}
+							
+							return true;
+						}
+					},
+					{
+						name: "dataModelName",
+						message: "Enter name of the data model",
+						default: "Book",
+						when: ({existingDataModelName}) => {
+							return !existingDataModelName;
+						},
+						validate: (name) => {
+							if(!name.match(/^([a-z]+)$/i)) {
+								return "A valid entity name must consist of letters only.";
+							}
+							return true;
+						}
+					}
```

Comment:
This is only about consistency: `"Enter name of the data model"`, but then `"A valid entity name must..."`. So, is it a `data model` or `entity`? If you're using `data model` - stick to it everywhere.